discovery-provider/plugins/notifications/src/processNotifications/mappers/trendingTrack.ts

```diff
@@ -114,11 +122,22 @@ export class TrendingTrack extends BaseNotification<TrendingTrackNotificationRow
     }
 
     if (
+      isLiveEmailEnabled &&
       userNotificationSettings.shouldSendEmail({
         receiverUserId: notificationReceiverUserId
       })
     ) {
-      // TODO: Send out email
+      const notification: AppEmailNotification = {
+        receiver_user_id: this.receiverUserId,
```

Comment:
nit, this is actually correct already but maybe use `notificationReceiverUserId` for extra clarity. totally up to u tho